client/js/components/procedure/StatementSegmentsList/StatementMeta/StatementMeta.vue

```diff
@@ -128,7 +128,7 @@
               text: Translator.trans('statement.date.authored')
             }"
             :value="localStatement.attributes.authoredDate ? localStatement.attributes.authoredDate : '-'"
```

Comment:
Shouldn't this be `model-value`?

Reply:
Done in https://github.com/demos-europe/demosplan-core/pull/3000/commits/171c2e45e02fcb725c7645149263e5849ff22f1d